runtime/compiler/runtime/JITServerStatisticsThread.cpp

```diff
@@ -112,10 +113,17 @@ static int32_t J9THREAD_PROC statisticsThreadProc(void * entryarg)
                avgCpuUsage = cpuUtil->getAvgCpuUsage();
                vmCpuUsage = cpuUtil->getVmCpuUsage();
                }
+            OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
```

Comment:
I don't know what the overhead of this is, but I believe it can be moved outside the loop, near `PORT_ACCESS_FROM_JAVAVM(vm);`. BTW do we need both `PORT_ACCESS` and `OMRPORT_ACCESS`?

Reply:
Yes, we need OMRPORT_ACCESS for ``omrstr_ftime_ex`` function